### DISCUSSION TITLE
Welcome to Universal Data Tool Discussions!

### DISCUSSION THREAD

**Discussion by seveibar** [UniversalDataTool/universal-data-tool#403] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.

**Comment by bobaekang** [comment]
Hi @seveibar. First off, thanks for creating UDT.

My team is currently evaluating UDT for an internal project and it looks really promising. However, we're a bit concerned that there hasn't been much activities lately. Is UDT no longer maintained/under active development?

**Reply by seveibar** [reply]
UDT isn't actively maintained at the moment, I'm not working as closely in the machine learning field.

**Reply by bobaekang** [reply]
Thanks for your answer!

**Comment by bona1507** [comment]
how to install udt on desktop
